xml/deployment_additional.xml

```diff
@@ -18,41 +18,59 @@
  <para>
   Once your cluster is ready, you may want to deploy additional software
   that is not installed on &productname; by default. This chapter provides
-  instructions on how to install and configure various tools and services.
+  instructions on how to install and configure &helm;, the &kube; package 
+  manager, and then how to install the &kube; Dashboard. If you intend to 
+  deploy &scf; on &productname;, &helm; is required.
  </para>
 
 <sect1 xml:id="installing.helm">
   <title>Deploying &helm; and &tiller;</title>
 <para>
-&helm; charts provide a standard way to deploy &kube; applications. While
-&tiller; (the &helm;'s server-side component) is not installed on &productname;
-by default, it can be installed by enabling the appropriate option in the Velum graphical
-interface. When the <guimenu>Install &tiller;</guimenu> option is enabled,
-&tiller; is installed as a container on a &productname; 2 cluster from the
-installation media.
+&helm; has two parts: &helm; is the client and &tiller; is the server component. 
+&helm; runs on your remote workstation that has access to your cluster, and 
+&tiller; is installed as a container on &productname; when you run &dashboard; 
+for the first 
+time. (See <xref linkend="sec.caasp.installquick.conf"/>.)
 </para>
 
+ <para>
+     You should match the &helm; version with the version of &tiller; 
```

Comment:
Helm can also help you with versions. Sometimes the versions are completely incompatible and you will get an error, but generally you would see:
helm is older than the server version here

~> helm version
Client: &version.Version{SemVer:"v2.7.2", GitCommit:"8478fb4fc723885b155c924d1c8c410b7a9444e6", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.8.2", GitCommit:"2.8.2", GitTreeState:"clean"}

newest release
~/Downloads> linux-amd64/helm version
Client: &version.Version{SemVer:"v2.9.0-rc1", GitCommit:"280863d2362345450125ce15c3db504471dde2cf", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.8.2", GitCommit:"2.8.2", GitTreeState:"clean"}

Reply:
Thanks Stef. I'll think about how to incorporate helm version. I don't work on CaaSP very much so I have to think a lot.